### PR TITLE
[zh-cn]: add translation for `DataTransferItem.getAsFileSystemHandle()`

### PR DESCRIPTION
**Pull request by skyclouds2001**
part of the #19988

### CODE REVIEW

File: files/zh-cn/web/api/datatransferitem/getasfilesystemhandle/index.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+title: DataTransferItem：getAsFileSystemHandle() 方法
+slug: Web/API/DataTransferItem/getAsFileSystemHandle
+l10n:
+  sourceCommit: 2b6bddfe281c0179fbde9c870f9de7c0dc3829e8
+---
+
+{{securecontext_header}}{{APIRef("File System API")}}{{SeeCompatTable}}
+
+{{domxref("DataTransferItem")}} 接口的 **`getAsFileSystemHandle()`** 方法返回一个 {{domxref('FileSystemFileHandle')}} 若拖动的项目是文件，或者返回一个 {{domxref( 'FileSystemDirectoryHandle')}} 若拖动的项目是目录。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/datatransferitem/getasfilesystemhandle/index.md:10]:
```suggestion
{{domxref("DataTransferItem")}} 接口的 **`getAsFileSystemHandle()`** 方法返回一个 {{domxref('FileSystemFileHandle')}}，若拖动的项目是文件，或者返回一个 {{domxref( 'FileSystemDirectoryHandle')}}，若拖动的项目是目录。
```

File: files/zh-cn/web/api/datatransferitem/getasfilesystemhandle/index.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+### 参数
+
+无。
+
+### 返回值
+
+一个 {{jsxref('Promise')}}。
+
+如果项目的 {{domxref("DataTransferItem.kind", "kind")}} 属性为 `"file"`，并且在 {{domxref("HTMLElement/dragstart_event", "dragstart")}} 或 {{domxref("HTMLElement/drop_event", "drop")}} 事件处理器中访问此项目，则返回的 promise 兑现一个 {{domxref('FileSystemFileHandle')}} 若拖动的项目是文件或一个 {{domxref ('FileSystemDirectoryHandle')}} 若拖动的项目是一个目录。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/datatransferitem/getasfilesystemhandle/index.md:26]:
```suggestion
如果项目的 {{domxref("DataTransferItem.kind", "kind")}} 属性为 `"file"`，并且在 {{domxref("HTMLElement/dragstart_event", "dragstart")}} 或 {{domxref("HTMLElement/drop_event", "drop")}} 事件处理器中访问此项目，则返回的 promise 兑现一个 {{domxref('FileSystemFileHandle')}}，若拖动的项目是文件；或一个 {{domxref ('FileSystemDirectoryHandle')}}，若拖动的项目是一个目录。
```

File: files/zh-cn/web/api/datatransferitem/getasfilesystemhandle/index.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
+---
+title: DataTransferItem：getAsFileSystemHandle() 方法
+slug: Web/API/DataTransferItem/getAsFileSystemHandle
+l10n:
+  sourceCommit: 2b6bddfe281c0179fbde9c870f9de7c0dc3829e8
+---
+
+{{securecontext_header}}{{APIRef("File System API")}}{{SeeCompatTable}}
+
+{{domxref("DataTransferItem")}} 接口的 **`getAsFileSystemHandle()`** 方法返回一个 {{domxref('FileSystemFileHandle')}}，若拖动的项目是文件，或者返回一个 {{domxref( 'FileSystemDirectoryHandle')}}，若拖动的项目是目录。
```
Comment (yin1999) [inline files/zh-cn/web/api/datatransferitem/getasfilesystemhandle/index.md:10]:
```suggestion
{{domxref("DataTransferItem")}} 接口的 **`getAsFileSystemHandle()`** 方法返回一个 {{domxref('FileSystemFileHandle')}}（若拖动的项目是文件），或 {{domxref( 'FileSystemDirectoryHandle')}}（若拖动的项目是目录）。
```

File: files/zh-cn/web/api/datatransferitem/getasfilesystemhandle/index.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+### 参数
+
+无。
+
+### 返回值
+
+一个 {{jsxref('Promise')}}。
+
+如果项目的 {{domxref("DataTransferItem.kind", "kind")}} 属性为 `"file"`，并且在 {{domxref("HTMLElement/dragstart_event", "dragstart")}} 或 {{domxref("HTMLElement/drop_event", "drop")}} 事件处理器中访问此项目，则返回的 promise 兑现一个 {{domxref('FileSystemFileHandle')}}，若拖动的项目是文件；或一个 {{domxref ('FileSystemDirectoryHandle')}}，若拖动的项目是一个目录。
```
Comment (yin1999) [inline files/zh-cn/web/api/datatransferitem/getasfilesystemhandle/index.md:26]:
```suggestion
如果项目的 {{domxref("DataTransferItem.kind", "kind")}} 属性为 `"file"`，并且在 {{domxref("HTMLElement/dragstart_event", "dragstart")}} 或 {{domxref("HTMLElement/drop_event", "drop")}} 事件处理器中访问此项目，则返回的 promise 兑现一个 {{domxref('FileSystemFileHandle')}}（若拖动的项目是文件）或 {{domxref ('FileSystemDirectoryHandle')}}（若拖动的项目是一个目录）。
```

File: files/zh-cn/web/api/datatransferitem/getasfilesystemhandle/index.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+否则，promise 会兑现 `null`。
+
+### 异常
+
+无。
+
+## 示例
+
+本示例使用 `getAsFileSystemHandle()` 方法返回{{domxref('FileSystemHandle', '文件句柄', '', 'nocode')}}用于删除的项目。
```
Comment (yin1999) [inline files/zh-cn/web/api/datatransferitem/getasfilesystemhandle/index.md:36]: 拖放

```suggestion
本示例使用 `getAsFileSystemHandle()` 方法返回一个表示放置的项目的{{domxref('FileSystemHandle', '文件句柄', '', 'nocode')}}。
```

**Review by yin1999 — APPROVED** [review]
Thank you @skyclouds2001

**Pull request merged**